addons/mail/static/src/components/message/message_tests.js

```diff
@@ -23,8 +23,20 @@ QUnit.module('message_tests.js', {
         this.createMessageComponent = async (message, otherProps) => {
             const MessageComponent = components.Message;
             MessageComponent.env = this.env;
+            let threadViewer;
+            if (!message.originThread) {
+                threadViewer = this.env.models['mail.thread_viewer'].create({
+                    thread: [['create', {id: 11, model: 'channel_channel'}]],
+                });
+                message.update({ originThread: [['link', threadViewer.thread]] })
+            } else {
+                threadViewer = this.env.models['mail.thread_viewer'].create({
+                    thread: [['link', message.originThread]],
+                });
+            }
```

Comment:
It would be better if the test used data from init messaging but I guess it would be ok to keep this at least until my PR improving it.

@alexkuhn could you check this PR?

Reply:
I don't like always making a thread viewer for any `Message` component test.
IMO tests that need to test `Message` in the context of a thread viewer should explicitly do so in the setup.

_(could argue tests that require a thread viewer should be a thread viewer test...)_